src/deploy/test_account_block.sql

```diff
@@ -312,6 +312,22 @@ BEGIN
   RETURN _result;
 END $$ LANGUAGE plpgsql;
 
+CREATE OR REPLACE FUNCTION maevsi_test.invoker_set (
+  _invoker_id UUID
+)
+RETURNS VOID AS $$
+BEGIN
+  SET LOCAL role = 'maevsi_account';
+  EXECUTE 'SET LOCAL jwt.claims.account_id = ''' || _invoker_id || '''';
+END $$ LANGUAGE plpgsql;
+
+CREATE OR REPLACE FUNCTION maevsi_test.invoker_unset ()
+RETURNS VOID AS $$
+BEGIN
+  SET LOCAL role = 'postgres';
```

Comment:
This needs to be 
```suggestion
  CALL maevsi_test.set_local_superuser();
```
since https://github.com/maevsi/sqitch/pull/159 which is awaiting your review.

Reply:
I've resolved the this merge conflict with the target branch for you, you can resolve this once read.